docs/querying/multi-value-dimensions.md

```diff
@@ -498,7 +498,7 @@ Having specs are applied at the outermost level of groupBy query processing.
 ## Disable GroupBy on multi-value columns
 
 You can disable the implicit unnesting behavior for groupBy by setting `groupByEnableMultiValueUnnesting: false` in your 
-[query context](query-context.md). In this mode, the groupBy engine will return an error instead of completing the query. This is a safety 
+[query context reference](query-context-reference.md). In this mode, the groupBy engine will return an error instead of completing the query. This is a safety 
```

Comment:
```suggestion
[query context](query-context-reference.md). In this mode, the groupBy engine will return an error instead of completing the query. This is a safety 
```